### PR TITLE
Impact report page base FE

### PR DESCRIPTION
**Pull request by siimonevans**
Base FE for new report page content type. There's a fair bit in here as we've needed to override a lot of existing layout (mostly streamfield containers/specific margins), but I've duplicated rather than modifying where appropriate.

Figma [here](https://www.figma.com/file/H5pQECphbbYsvSWtag3OrCWC/Torchbox?node-id=1516%3A4020&t=6vIPpjIpDjV4W2ra-0).

Screenshot 2022-12-14 at 10 45 56

Screenshot 2022-12-14 at 10 45 51

Screenshot 2022-12-14 at 10 45 51a

### CODE REVIEW

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/impact_report_heading_block.html [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-    <div style="padding-top: 75px;">
-        <b>{{ value.short_heading }}</b>
-    </div>
+<div class="report-section">
+    {% image value.image fill-1792x597 class="report-section__image" %}
 
-    <div>
-        {% image value.image fill-1000x300 %}
+    <div class="report-section__container">
+        <div>
```
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/impact_report_heading_block.html:7]: Q: is this div needed?

File: tbx/static_src/sass/components/_in-page-nav.scss [hunk 2]
```diff
@@ -156,4 +156,61 @@
             font-size: map-get($large-font-sizes, s);
         }
     }
+
+    &--report {
+        margin-left: 8.33vw;
```
Comment (chris-lawton) [inline tbx/static_src/sass/components/_in-page-nav.scss:161]: This is assigned to a var if helpful - https://github.com/torchbox/wagtail-torchbox/blob/master/tbx/static_src/sass/abstracts/_variables.scss#L111

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/impact_report_heading_block.html [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+            <p class="report-section__short-heading" id="{{ value.short_heading|slugify }}" data-service-section>
+                {{ value.short_heading }}
+            </p>
+        </div>
+        <h2 class="report-section__heading">{{ value.heading|richtext }}</h2>
```
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/impact_report_heading_block.html:12]: Q: Guessing these need to have a richtext filter so the highlighted word can be styled?

**Review by chris-lawton — APPROVED** [review]

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/impact_report_heading_block.html (hunk 1 above)
Comment (siimonevans) [reply tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/impact_report_heading_block.html:7]: Yeah, bit weird but I think so - I needed it to make the grid work correctly while also allowing the `<p>` to remain inline for the top-border/max-width.

File: tbx/static_src/sass/components/_in-page-nav.scss (hunk 2 above)
Comment (siimonevans) [reply tbx/static_src/sass/components/_in-page-nav.scss]: Ah nice one, thanks! Updated a couple of instances 👍

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/impact_report_heading_block.html (hunk 3 above)
Comment (siimonevans) [reply tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/impact_report_heading_block.html:12]: Yeah, it looks like that's how it's done across the site - allows output something like `<p>My text <span>goes here</span></p>`.

**Pull request merged**